UniswapV3Adapter/contracts/interfaces/IAdapter.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.htm